Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -48,7 +52,7 @@ There are usually two release engineers working at any given time. It goes witho
 <br/>	*It starts after all opened issues had been adressed and can take several cycles until QA's approval*
 
 1. Triger a new release build in the `#ios-build` channel
-1. Obtain the release notes form the Product Manager  and update them in the [AppStoreConnect](https://appstoreconnect.apple.com)
+1. Obtain the release notes from the Product Manager and update them in the [AppStoreConnect](https://appstoreconnect.apple.com)
 1. Enable the new release version in [AppStoreConnect](https://appstoreconnect.apple.com).
 1. Perform a quick exploratory test on the TestFlight build to make sure everything looks okay. (e.g. verifying that DigitalTwin Assets are visible and are not dropped due to Git LFS issues) ❗️ NOTE: Remember to submit compliance info for that build.
```

Comment:
Shouldn't we have more clarity on what is involved in "compliance info" submission?

Reply:
That's a good question (you'll probably need to ask Andreea or Lilla for more info)